### DISCUSSION TITLE
Blog Post Navigation

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/site#734] [General]
# Blog Post Navigation

The post nav module generates the menus below the post content, to navigate to previous and next posts of same sections.

https://hbstack.dev/modules/blog/post-nav/

**Comment by diogoguilhen** [comment]
in the main page have a post image with
<svg aria-hidden="true" class="bi bi-file-richtextbi bi-file-richtext hi-svg-inline text-primary" fill="currentColor" height="2.25em" viewBox="0 0 16 16" width="2.25em" xmlns="http://www.w3.org/2000/svg">
  <path d="M7 4.25a.75.75 0 1 1-1.5 0 .75.75 0 0 1 1.5 0m-.861 1.542 1.33.886 1.854-1.855a.25.25 0 0 1 .289-.047l1.888.974V7.5a.5.5 0 0 1-.5.5H5a.5.5 0 0 1-.5-.5V7s1.54-1.274 1.639-1.208M5 9a.5.5 0 0 0 0 1h6a.5.5 0 0 0 0-1zm0 2a.5.5 0 0 0 0 1h3a.5.5 0 0 0 0-1z"></path>
  <path d="M2 2a2 2 0 0 1 2-2h8a2 2 0 0 1 2 2v12a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2zm10-1H4a1 1 0 0 0-1 1v12a1 1 0 0 0 1 1h8a1 1 0 0 0 1-1V2a1 1 0 0 0-1-1"></path>
</svg>


but it is withou link did u know where i can fix it?

**Reply by diogoguilhen** [reply]
sry the image didn't up  it's look like a paper blue: https://productcommunity.com.br/pt-br/
u can see here in blue with "11" posts

**Reply by razonyang** [reply]
It's inteneded, since we don't have a page to show all the posts.
